github/orgs_actions_allowed.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2020 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+)
+
+// ActionsAllowed represents selected actions that are allowed in an organization
```

Comment:
Please make all godoc-style comments complete sentences by adding a period to the end:

```suggestion
// ActionsAllowed represents selected actions that are allowed in an organization.
```